docker-accumulo/Makefile

```diff
@@ -1,10 +1,54 @@
+all: start
 
-all:
+IMAGE_NAME := accumulo-$(shell whoami)
+CONTAINER_NAME := accumulo-$(shell whoami)
+OUT := out/
 
-CONTAINERTAG := accumulo
+HADOOP_IP_FILE := ../docker-hadoop/build/hostname
+ZOOKEEPER_IP_FILE := ../docker-zookeeper/build/hostname
 
-build: Dockerfile
-	docker build -t $(CONTAINERTAG) .
+HADOOP_IP := $(shell cat $(HADOOP_IP_FILE))
+ZOOKEEPER_IP := $(shell cat $(ZOOKEEPER_IP_FILE))
+
+include ../Makefile.options
+
+build := build/image
+build: $(build)
+$(build): Dockerfile
+	mkdir -p build/
+	docker build -t $(IMAGE_NAME) .
+
+cid_file := build/container_id
+
+$(cid_file): $(build) $(HADOOP_CONFIG_DIR)/core-site.xml
+	docker create							\
```

Comment:
move lines 32-34 in this file to line 24 here (and then delete the install rule)